### ISSUE TITLE
Vertical scrollbar issue on Windows

### ISSUE THREAD

**Issue by MJRomka** [joeattardi/emoji-button#61]
Vertical scrollbar on Windows takes some part of area width that leads to horizontal scrollbar appearing which looks ugly
![scrollbar_emoji](https://user-images.githubusercontent.com/3237392/83190492-2a1e6980-a13b-11ea-8637-28774002d450.png)

**Comment by MJRomka** [comment]
You can consider to hide scrollbars on Windows:

```
.emoji-picker__emojis::-webkit-scrollbar,
.emoji-picker__container.search-results::-webkit-scrollbar {
    display: none;
}
@-moz-document url-prefix() {
    .emoji-picker__emojis,
    .emoji-picker__container.search-results {
        scrollbar-width: none;
    }
}
```

**Comment by joeattardi** [comment]
That's definitely not good. I think instead of hiding the scroll bar on Windows, I can try adding horizontal padding to the picker to make room for the scroll bar so that there is no horizontal scrolling.

I am on Mac so never saw this issue. Thanks!

**Comment by joeattardi** [comment]
I released version 3.1.1 today which hopefully will solve this problem. I don't have a Windows machine so please give it a try and let me know if you still have this issue. Thanks!

**Comment by MJRomka** [comment]
Could you update version in the demo https://emoji-button.js.org/ ? The issue persists there

**Comment by RichardDern** [comment]
Same issue here, on Safari + macOS.

**Comment by joeattardi** [comment]
Darn, looks like I didn't fix the issue. Will take another look at this. Sorry about that!

**Comment by mdunicorn** [comment]
The reason for this, are emojis that consist of more than one character. Windows will not render them correctly:

![image](https://user-images.githubusercontent.com/1384790/87154333-4e3ea000-c2ce-11ea-9afa-a5301c43d191.png)

For example, [the pilot emoji](https://emojipedia.org/pilot/) is rendered like 

![image](https://user-images.githubusercontent.com/1384790/87154421-6f06f580-c2ce-11ea-8ce4-959235316cb2.png)

instead of

![image](https://user-images.githubusercontent.com/1384790/87154575-ba210880-c2ce-11ea-9a3f-c728edd1b97a.png)

(on Apple)

**Comment by dannielhugo** [comment]
I'm also having this issue. I'm using Kubuntu 18.04

**Comment by joeattardi** [comment]
If the cause is the emojis not rendering right, as is shown in @mdunicorn's comment, there's not really anything that can be done other than (1) upgrade your OS, or (2) set the `emojiVersion` property to an older emoji version that doesn't have the incompatible emojis.

**Comment by joeattardi** [comment]
I made a change that will alleviate the scrollbar, at least. If the emoji is rendered as two separate emojis because it's not supported by the OS, the overflowing content will be cut off so that it fits within the grid cell. Thus it will look "cut off", but you should no longer see the scrollbar.

Best compromise I could come up with 😄

**Comment by joeattardi** [comment]
BTW, this fix will be released with version 4.0.0.
